AUTHORS.rst

```diff
@@ -0,0 +1,13 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2019 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+
+Contributors
+============
+
+- Pablo Panero
+- Guillaume Viger
```

Comment:
minor (might as well do): according to @lnielsen other PR, authors should be in alphabetical order of first name I believe.

Reply:
:+1: